src/singlylinkedlist.c

```diff
@@ -176,7 +176,7 @@ LIST_ITEM_HANDLE singlylinkedlist_get_head_item(SINGLYLINKEDLIST_HANDLE list)
         LIST_INSTANCE* list_instance = list;
 
         /* Codes_SRS_LIST_01_008: [singlylinkedlist_get_head_item shall return the head of the list.] */
-        /* Codes_SRS_LIST_01_010: [If the list is empty, singlylinkedlist_get_head_item_shall_return NULL.] */
```

Comment:
>If the list is empty, singlylinkedlist_get_head_item_shall_return NULL. [](http://example.com/codeflow?start=35&length=71)

The spec in the .md needs to be updated to

"If the list is empty, singlylinkedlist_get_head_item_shall_return NULL."
Of course with backticks and all.

Reply:
Done